.github/ISSUE_TEMPLATE.md

```diff
@@ -33,11 +33,12 @@ Check one of the following options with "x" -->
 
 ## Environment
 
-<pre><code>
-For Tooling issues:
-- Node version: XX  <!-- run `node --version` -->
-- Platform:  <!-- Mac, Linux, Windows -->
+#### For Tooling issues:
+<!-- Run `node --version` -->
+- Node version: XX
+
+<!-- Mac, Linux, Windows -->
+- Platform:
 
-Others:
+#### Others:
 <!-- Anything else relevant?  Operating system version, IDE, package manager, ... -->
-</code></pre>
```

Comment:
I think it looks cleaner without the `<code>` and `<pre>` tag. And to avoid rendering the comments, I had to put them in different lines.